Demo/src/main/java/com/braintreepayments/demo/PayPalMessagingFragment.kt

```diff
@@ -26,12 +26,11 @@ class PayPalMessagingFragment : BaseFragment(), PayPalMessagingListener {
 
         val payPalMessagingRequest = PayPalMessagingRequest(
             amount = 2.0,
-            placement = null,
+            pageType = null,
             offerType = PayPalMessagingOfferType.PAY_LATER_LONG_TERM,
             buyerCountry = "US",
             logoType = PayPalMessagingLogoType.PRIMARY,
-            textAlignment = PayPalMessagingTextAlignment.CENTER,
-            color = null
```

Comment:
Is `color` no longer needed?

Reply:
`Color` is still `required`; in fact, it's now `required`. I had previously removed it so it would `default` to the value specified in the `PayPalMessagingRequest` constructor. I've added it back